libs/watt/package/modal/watt-modal.service.ts

```diff
@@ -112,9 +112,3 @@ export class WattModalService {
     });
   }
 }
-
-@NgModule({
-  imports: [MatDialogModule],
-  providers: [WattModalService],
-})
-export class WattModalModule {}
```

Comment:
issue: CI is failing with some errors related to this. Also the change might cause issues in many places where a modal is used, so we'll need to check the different ways a modal is shown (instantiated via a service, via a route, used directly in a template, etc.) to make sure it all still works

Reply:
They are fixed now. I think we are okay. I can try to deploy to dev02, and click more around?